src/reactionsystem_conversions.jl

```diff
@@ -1,7 +1,7 @@
 ### ODE & SDE Assembly ###
 
 """
-    oderatelaw(rx; combinatoric_ratelaw=true)
+    oderatelaw(rx; combinatoric_ratelaw = true)
```

Comment:
My impression is that we now have changed to having space in between `=` and whatever is on each side. If this is not the case, feel free to correct me though.

Reply:
(there is a couple of changes like this throughout the file)